Fix group more links.

dkinzer opened the pull request:
Ref: https://github.com/NuCivic/healthdata/issues/466
Ref: https://github.com/NuCivic/healthdata/issues/470
Ref: https://github.com/NuCivic/healthdata/issues/437

Acceptance:
- [ ] Group members page does not include "more" link back to itself.
- [ ] Group members block adds custom "more" link the works (i.e. doesn't go to page
  not found)
- [ ] When looking at the resource node form, any html entity in the datasets field (which is a list of dataset titles) is decoded.
- [ ] Disables og_extras_members view.

dkinzer committed:
Fix group more links.

Ref: https://github.com/NuCivic/healthdata/issues/466

Acceptance:

- [ ] Member page does not include "more" link back to itself.
- [ ] Members block adds custom "more" link the works (i.e. doesn't go to page
  not found)

nucivicmachine commented:
QA site created: http://healthdata-466-fix-group-more-links.dkan_dataset.qa.nuamsdev.com

dkinzer committed:
Fix apostrophe show up as html entity in selection.

Ref: https://github.com/NuCivic/healthdata/issues/470

dkinzer commented:
I'm piggy backing fix for: https://github.com/NuCivic/healthdata/issues/470 here as well.

nucivicmachine commented:
QA site created: http://healthdata-466-fix-group-more-links.dkan_dataset.qa.nuamsdev.com

nucivicmachine commented:
QA site created: http://healthdata-466-fix-group-more-links.dkan_dataset.qa.nuamsdev.com

nucivicmachine commented:
QA site created: http://healthdata-466-fix-group-more-links.dkan_dataset.qa.nuamsdev.com

dkinzer committed:
Fix bug caused by og_extras_members view enabled.

When both dkan_og_extras_members and og_extras_members are enabled there is that
is caused by both views trying to do the same thing in the same path.  This
commit disables og_extras_members so that the bug is fixed.

Ref: https://github.com/NuCivic/healthdata/issues/466

nucivicmachine commented:
QA site created: http://healthdata-466-fix-group-more-links.dkan_dataset.qa.nuamsdev.com

dkinzer commented:
@dafeder I added bit to disable the `og_extras_members` view on installation or update.

dafeder commented:
I've merged, and am adding a call to   _dkan_dataset_disable_og_extras_group_members_view();
 to the DKAN install script.
